packages/documentation/src/pages/getting-started/common-tasks/routing.mdx

```diff
@@ -0,0 +1,43 @@
+# Routing
+
+## Proxy Config
+
+The proxy handles incoming URL requests to va.gov and is managed through a [proxy config](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/revproxy-vagov/vars/react_routes.yml) file.
+
+## React Router
+
+To use client side routing such as react-router, the `react_routes` section must be configured to ignore the route.
+For example, a React app running at `www.va.gov/appPath` would need this added:
+
+```yml
+  - path: /appPath
+    protected: no
+```
+
+
+## Redirects
```

Comment:
```suggestion
## Server-side Redirects
```
We have a lot of client side redirects so it would be good to differentiate. I would rather not document the client side redirects behavior right now since we haven't settled on a single approach and we want to discourage client side redirects.